keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -39,7 +41,9 @@ class VerificationRunner(
       }
 
       statuses.firstOutstanding?.let { verification ->
-        start(verification, imageFinder.getImages(context.deliveryConfig, context.environmentName))
+        enforcer.withVerificationLease(this) {
```

Comment:
Why do we need this here in addition to the one in `CheckScheduler`?

Reply:
Because verifications aren't executed by the `CheckScheduler.checkResources` function, they're executed by the `CheckScheduler.verifyEnvironments` function, which is a different loop.